comictaggerlib/taggerwindow.py

```diff
@@ -1792,36 +1852,42 @@ def identify_and_tag_single_archive(
                 if self.config[0].Issue_Identifier__auto_imprint:
                     md.fix_publisher()
 
-                if not ca.write_metadata(md, self.save_data_style):
-                    match_results.write_failures.append(
-                        Result(
-                            Action.save,
-                            Status.write_failure,
-                            ca.path,
-                            online_results=matches,
-                            match_status=MatchStatus.good_match,
+                # Save each style
+                for style in self.save_data_styles:
+                    if not ca.write_metadata(md, style):
+                        match_results.write_failures.append(
+                            Result(
+                                Action.save,
+                                Status.write_failure,
+                                ca.path,
+                                online_results=matches,
+                                match_status=MatchStatus.good_match,
+                            )
```

Comment:
There should be one result per comic archive and use these attributes, just like how the cli does
```python
    tags_deleted: list[str] = dataclasses.field(default_factory=list)
    tags_written: list[str] = dataclasses.field(default_factory=list)
```

Reply:
I believe this is correct now? I didn't see where any `tags_deleted` would come from?